components/automate-ui/src/app/pages/team-details/team-details.component.ts

```diff
@@ -333,8 +299,6 @@ export class TeamDetailsComponent implements OnInit, OnDestroy {
   }
 
   dropdownDisabled(): boolean {
-    return Object.values(this.projects).length === 0 ||
-      this.teamProjectsLeftToFetch.length !== 0 ||
-      this.saving;
+    return Object.values(this.projects).length === 0 || this.saving;
```

Comment:
❓ I am aware that this was copied, but -- isn't there a more obvious way to check if `this.projects` is `{}`? That, to me, seems like the only wayt that `Object.values(this.projects).length` could be `0`. 🤔